app/models/import/import_users.rb

```diff
@@ -77,4 +75,15 @@ def error(job, exception)
     import = Import::Import.find(import_id)
     import.update_attribute(:progress, -1)
   end
+
+  def add_new_admin_tag(user, tag_type, tag_list)
```

Comment:
This is duplicate code. Moving it to a class method on `Admin::Tag` seems like the best approach.  
You can call it `Admin::Tag.add_and_assign_tags(taggable, tag_type, tag_list)`.